### PR TITLE
Puppet5 fixes

### PR DESCRIPTION
**Pull request by rahushen**
Made fixes to issues seen in puppet5. Errors seen in the tests were because `remove_all_vrfs` was silently failing to do it's job. Should now be fixed. 

All failed/errored tests were retested with puppet5 and latest puppet and are passing.

### CODE REVIEW

File: tests/beaker_tests/cisco_command_config/test_command_config.rb [hunk 1]
```diff
@@ -193,7 +193,7 @@ def test_set_get
   logger.info('* check config')
   on(agent, cmd_prefix + "test_get='incl loopback1'")
   fail_test("TestStep :: set/get :: FAIL\nstdout:\n#{stdout}") unless
-    stdout[/^interface loopback1/]
+    stdout[/^?\n?interface loopback1/]
```
Comment (mikewiebe) [inline tests/beaker_tests/cisco_command_config/test_command_config.rb]: Can you add some sample output from both agent versions that this is trying to match?  Also add comments to make it clear that we need to match two versions of output.

File: tests/beaker_tests/lib/utilitylib.rb [hunk 2]
```diff
@@ -1463,8 +1463,8 @@ def remove_all_vlans(agent, stepinfo='Remove all vlans & bridge-domains')
 end
 
 def remove_all_vrfs(agent)
-  found = test_get(agent, "incl 'vrf context' | excl management").split("\n")
-  found.map! { |cmd| "no #{cmd}" if cmd[/^vrf context/] }
+  found = test_get(agent, "incl 'vrf context' | excl management").split('\\n')
```
Comment (mikewiebe) [inline tests/beaker_tests/lib/utilitylib.rb]: Same comment here.

File: tests/beaker_tests/cisco_command_config/test_command_config.rb (hunk 1 above)
Comment (rahushen) [inline tests/beaker_tests/cisco_command_config/test_command_config.rb]: Done.

File: tests/beaker_tests/lib/utilitylib.rb (hunk 2 above)
Comment (rahushen) [inline tests/beaker_tests/lib/utilitylib.rb]: Done.

**Comment by mikewiebe** [comment]
👍

**Pull request merged**